src/neapolitan/templatetags/neapolitan.py

```diff
@@ -55,7 +55,8 @@ def object_list(objects, fields):
         {
             "object": {f: getattr(object, f) for f in fields},
             "fields": [
-                object._meta.get_field(f).value_to_string(object) for f in fields
+                {"name": field, "value": str(getattr(object, field))}
+                for field in fields
```

Comment:
This seems like a win, but I'm a little confused if we should look for a `value` attribute existing? 

:eyes: I suspect this is one of those changes that when you refresh, you see something, so then you change it, and this was what made it display correctly.

Reply:
Not sure why I went the way I did, but I wonder if just keeping it the way it is in the OG code and just adding the "name" would be the way to go:

```suggestion
                {"name": field, "value": object._meta.get_field(field).value_to_string(object)}
                for field in fields
```